implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -13,6 +15,12 @@ pub enum AppError {
     #[error("invalid argument")]
     InvalidArgument,
 
+    #[error("i/o error")]
+    IOError(std::io::Error),
```

Comment:
In Rust it's considered better style to camel-case abbreviations (avoids `XMLHttpRequest`-style situations)
```suggestion
    IoError(std::io::Error),
```

Reply:
More broadly we'll probably want to use something like `anyhow` for the CLI, but that can be addressed later.